userguide/plugins.rst

```diff
@@ -223,6 +223,10 @@ set and :guilabel:`DELETE` is clicked.
 Create a Plugin
 ---------------
 
+
+Before creating a plugin, read the
+`LICENSE.md <https://github.com/ix-plugin-hub/documentation/blob/master/LICENSE.md>`__
```

Comment:
Add period to end of line, but move the license section to after the introduction.